### PR TITLE
Add token-transfer to blockstack-cli

### PR DESCRIPTION
**Pull request by lgalabru**
This PR is for unblocking @yknl, working on implementing wire format in TS.

### CODE REVIEW

**Comment by jcnelson** [comment]
Please add some unit-tests for it and we'll get this merged :+1:

**Comment by lgalabru** [comment]
@jcnelson, yeah, wanted to share the branch asap and was working on testing in parallel.

File: src/blockstack_cli.rs [hunk 1]
```diff
@@ -286,6 +293,36 @@ fn handle_contract_call(args: &[String], version: TransactionVersion) -> Result<
...
+    }
+    if args.len() < 5 {
+        return Err(CliError::Message(format!("Incorrect argument count supplied \n\nUSAGE:\n {}", CALL_USAGE)))
+    }
+    let sk_origin = StacksPrivateKey::from_hex(&args[0])?;
+    let fee_rate = args[1].parse()?;
+    let nonce = args[2].parse()?;
+    let recipient_address = StacksAddress::from_string(&args[3]).ok_or("Failed to parse contract address")?;
+    let amount = &args[4].parse()?;
+    let memo = TokenTransferMemo([0u8; 34]);
```
Comment (jcnelson) [inline src/blockstack_cli.rs]: This should probably be a CLI argument.  Lots of token-transfers in Stacks v1 use this field.

Comment (lgalabru) [reply src/blockstack_cli.rs]: Addressed with https://github.com/blockstack/blockstack-core/pull/1259/commits/66cdef1087b0db9208dd01fa7ff47bca2ad67903

**Comment by jcnelson** [comment]
LGTM!  Just make sure CircleCI passes first, and then feel free to merge :+1:

**Comment by jcnelson** [comment]
Whoops, looks like the `token-transfer` test is broken @lgalabru.  Please fix this first before merging.

**Pull request merged**